xml_feed/xml_feed.py

```diff
@@ -110,17 +149,33 @@ def trigger(self, previous_result, config, names, tags, run_all, rate_limit,
         if not config['collection'].endswith('/'):
             config['collection'] = '{}/'.format(config['collection'])
 
-        config = self.parse_config(config)
+        options = Options(config)
+
+        sanitized_config = dict(
+            (k,v) for k,v in config.iteritems()
+            if k not in CONFIG_FIELDS_TO_REMOVE)
+        config_message = self.parse_config(sanitized_config)
 
-        raw_feed = self._download_feed(config.url)
-        for article in self._parse_articles(raw_feed):
-            pod_path = '{}{}.html'.format(config.collection, article.slug)
+        raw_feed = self._download_feed(config_message.url)
+        for article in self._parse_articles(raw_feed, options):
+            removed_duplicate_dots = re.sub(r'([\.]{2,})', '.', article.slug)
```

Comment:
You can remove the `(` and `)`, but that should do the trick.

Reply:
Done :)